packages/flutter/lib/src/rendering/mouse_tracker.dart

```diff
@@ -367,40 +257,41 @@ abstract class BaseMouseTracker extends ChangeNotifier {
     return _hitTestResultToAnnotations(hitTest(globalPosition));
   }
 
-  /// A callback that is called on the update of a device.
-  ///
-  /// This method should be called only by [BaseMouseTracker], each time when the
-  /// relationship between a device and annotations has changed.
-  ///
-  /// By default the [handleDeviceUpdate] does nothing effective. Subclasses
-  /// should override this method to first call to their inherited
-  /// [handleDeviceUpdate] method, and then process the update as desired.
-  ///
-  /// The update can be caused by two kinds of triggers:
-  ///
-  ///   * Triggered by the addition, movement, or removal of a pointer. Such
-  ///     calls occur during the handler of the event, indicated by
-  ///     `details.triggeringEvent` being non-null.
-  ///   * Triggered by the appearance, movement, or disappearance of an annotation.
-  ///     Such calls occur after each new frame, during the post-frame callbacks,
-  ///     indicated by `details.triggeringEvent` being null.
-  ///
-  /// Calling of this method must be wrapped in `_deviceUpdatePhase`.
-  @protected
-  @mustCallSuper
-  void handleDeviceUpdate(MouseTrackerUpdateDetails details) {
+  // A callback that is called on the update of a device.
+  //
+  // An event (not necessarily a pointer event) that might change the relationship
+  // between mouse devices and [MouseTrackerAnnotation]s is called a _device
+  // update_. This method should be called at each such update.
+  //
+  // The update can be caused by two kinds of triggers:
+  //
+  //   * Triggered by the addition, movement, or removal of a pointer. Such
+  //     calls occur during the handler of the event, indicated by
+  //     `details.triggeringEvent` being non-null.
+  //   * Triggered by the appearance, movement, or disappearance of an annotation.
+  //     Such calls occur after each new frame, during the post-frame callbacks,
+  //     indicated by `details.triggeringEvent` being null.
+  //
+  // Calls of this method must be wrapped in `_deviceUpdatePhase`.
+  void _handleDeviceUpdate(_MouseTrackerUpdateDetails details) {
```

Comment:
Technically this is a breaking change to the API: I'm guessing it's not used much, however.

Reply:
Yeah we're now defining breaking changes by whether it breaks flutter/tests or g3. If it doesn't then it's fine to remove symbols.

Reply:
Sorry, I was unclear: I know it doesn't meet our definition of a "breaking change", I was just pointing out that it does break the API if anyone were using it, in case you didn't mean to break it.